packages/common/src/authentication/AuthContext.tsx

```diff
@@ -108,11 +104,12 @@ export const AuthProvider: FC = ({ children }) => {
   const i18n = IntlUtils.useI18N();
   const defaultLanguage = IntlUtils.useDefaultLanguage();
   const { mutateAsync, isLoading: isLoggingIn } = useGetAuthToken();
-  const { mutateAsync: getStores } = useStores();
   const authCookie = getAuthCookie();
   const [cookie, setCookie] = useState<AuthCookie | undefined>(authCookie);
   const [error, setError] = useLocalStorage('/auth/error');
   const storeId = cookie?.store?.id ?? '';
+  const { mutateAsync: getStores } = useUserDetails();
```

Comment:
changed to a `useQuery` then back to a mutation - the auth header has to be set when querying, and loading the store list before login is a problem for that!